release.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -euo pipefail
+
+NEW_VERSION="$(date +%Y.%m.%d)"
+OLD_VERSION="$(git tag --list | tail -n 1)"
+
+if [ "${NEW_VERSION}" = "${OLD_VERSION}" ]; then
+    echo "Nothing to release"
+    exit 0
+fi
+
+# Make sure local fork is updated
+git fetch -p --all
+git checkout master
+git merge --ff-only upstream/master
+
+# Bump version number
+echo "${NEW_VERSION}" > VERSION
+
+# Generate the package
+make package-clean package
+
+# Sanity check Pulp Smash packages on both Python 2 and Python 3
+for python in python{2,3}; do
+    virtualenv -p ${python} pulpsmash-${python}
```

Comment:
Ok, this really is my last suggestion! How about using `mktmp --directory` when choosing where to make a virtualenv? There shouldn't be an issue with making directories named `pulpsmash-python2` or `pulpsmas-python3`, so this isn't the most important thing in the world. All the same... it does seem proper.

```sh
for python in python{2,3}; do
    env="$(mktemp -d)"
    virtualenv --python "$python" "$env"
```